tools/test/run-test-file.ts

```diff
@@ -0,0 +1,40 @@
+import { spawn } from "child_process";
+import fs from "fs";
+import path from "path";
+
+const currentFile = process.argv[2];
+const packageJsonPath = findNearestPackageJson(currentFile);
```

Comment:
There is a library that we use elsewhere called `find-up`, which can be used for this. E.g. https://github.com/cloudflare/workers-sdk/blob/cfde0587c40cbcdfcb5e7981e8105e1b99f031fa/packages/wrangler/src/pages/utils.ts#L50

Reply:
Great, I'll use that instead then.

Reply:
https://github.com/cloudflare/workers-sdk/pull/5978/commits/8489118fae0c4d694d9c641ea2d297ae6fabdf95